### DISCUSSION TITLE
HUSD Funds not arrived at Solana pending at "Awaiting completion on Solana"

### DISCUSSION THREAD

**Discussion by somanyee** [wormhole-foundation/wormhole#241] [Problems / bugs] [answered]
Hi admin,
When doing HUSD <> wHUSD, funds stuck on Wormhole with "Awaiting completion on Solana" message. The funds were removed from my ETH wallet and were not deposited in my Solana Wallet.
Here is the ETH txid:
0x342273414cccded9623b32900ea9dd6b02c4fe4bfd491c455cdfcf5d1d1232d0

MyWallet Address:
GzP1BkWa7LnC71fgJ1iRbu6x1VTtveuAofFXkao4oiYT

Could you please resolve the issue?
Please let me know if you need any additional information.
Thank you.

**Comment by hendrikhofstadt** [comment, marked as answer]
Confirmed in https://explorer.solana.com/tx/4zsDGM8BUxtahbkVvy9pgfZMXjJMr4YywZbhN3V6dQCs72Q8QJvYhVE8LqwsaZmKFEvNPbJxoSBkqbpC5iNC7Qfj
